app/src/main/java/com/example/parkingsystem/mvp/model/ParkingModel.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.parkingsystem.mvp.model
+
+import com.example.parkingsystem.mvp.contract.ParkingContract
+import com.example.parkingsystem.util.Constants
+
+class ParkingModel : ParkingContract.Model {
+    private val parkingSize = Constants.PARKING_SIZE_DEFAULT
```

Comment:
if you are using this constany only here, you could put this const into the companion obj